components/sealed/sealed-catalog-item.tsx

```
@@ -81,37 +58,38 @@ const SealedCatalogItem = ({ product }: Props) => {
-            <div
-              className="mb-2 mt-1 flex w-full cursor-pointer flex-col items-center justify-center gap-1 transition-opacity hover:opacity-80 md:flex-row md:justify-start"
-              onClick={handleClick}
-            >
-              <div className="text-xs md:hidden">
-                {getVendorNameBySlug(product.vendor)}
-              </div>
+            </div>
+            <div className="flex flex-wrap items-center gap-1 text-[0.65rem] font-medium capitalize text-primary">
+              {product.tags &&
+                product.tags.map((tag: string) => (
+                  <div className="rounded bg-primary/10 px-1.5 py-0.5">
```

Biome (2.1.2)

[error] Missing key property for this element in iterable.

The order of the items may change, and having a key can help React identify which item was moved.
Check the React documentation. 

(lint/correctness/useJsxKeyInIterable)